contracts/MetaCoin.sol

```diff
@@ -1,44 +0,0 @@
-pragma solidity ^0.4.2;
-
-import "@gnosis.pm/gnosis-core-contracts/contracts/Utils/Math.sol";
-import "@gnosis.pm/gnosis-core-contracts/contracts/Events/EventFactory.sol";
-import "@gnosis.pm/gnosis-core-contracts/contracts/Tokens/EtherToken.sol";
-import "@gnosis.pm/gnosis-core-contracts/contracts/Oracles/CentralizedOracleFactory.sol";
-import "@gnosis.pm/gnosis-core-contracts/contracts/Oracles/UltimateOracleFactory.sol";
-import "@gnosis.pm/gnosis-core-contracts/contracts/MarketMakers/LMSRMarketMaker.sol";
-import "@gnosis.pm/gnosis-core-contracts/contracts/Markets/StandardMarketFactory.sol";
```

Comment:
This is tricky. I've included all these imports in a contract in this repo so that Truffle picks them up from the gnosis-contracts dependency, and because developing with TestRPC is very convenient. It would be most ideal to have the migrations from the dependency run instead of the stuff in this repo, but I'm not sure how to accomplish this. Doing so also causes the build artifacts to appear with synchronized network information, but ideally, this could be done without having an extraneous contract like this. I dunno, it's tricky.